### PR TITLE
PR 3 for thirs commit

### PR DESCRIPTION
**Pull request by EldadHellerman**

### CODE REVIEW

**Review by EldadHellerman — COMMENTED** [review]
this pull request is awesome!

**Comment by mentor-tara** [comment]
Looks like something is wrong :astonished:	Please see the CI checks below for more information. Click the "Details" link for more information as shown in this screenshot.

Screenshot of CI checks[](error)

**Review by EldadHellerman — COMMENTED** [review]
comments are now allowed in JSON!

**Comment by EldadHellerman** [comment]
JSON is still the best

**Review by EldadHellerman — COMMENTED** [review]
reviewing changes is cool

**Review by mentor-tara — APPROVED** [review]
### You're good. No mines hit. Merge this PR to continue the game.

This is the current status:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   | 1 | 2 | :pirate_flag: | :pirate_flag: | :pirate_flag: |
| **2** | 1 | 2 | 2 | :pirate_flag: | 2 | 2 | 3 | 2 |
| **3** | :pirate_flag: | 2 | :pirate_flag: | 2 | 1 | 0 | 0 | 0 |
| **4** | 1 | 2 | 1 | 1 | 0 | 0 | 0 | 0 |
| **5** | 0 | 0 | 0 | 0 | 0 | 0 | 1 | 1 |
| **6** | 0 | 0 | 0 | 1 | 1 | 1 | 2 | :pirate_flag: |
| **7** | 0 | 1 | 1 | 2 | :pirate_flag: | 1 | 2 | :pirate_flag: |
| **8** | 0 | 1 | :pirate_flag: | 2 | 1 | 1 | 1 | 1 |

Mines remaining: 0
Moves made: 24[](clear-H8)

File: commands.json [hunk 1]
```diff
@@ -10,6 +10,20 @@
     "clear F8",
     "clear G8",
     "clear B3",
-    "flag A3"
+    "flag A3",
+
```
Comment (EldadHellerman) [inline commands.json:14]: i like the newline here

File: commands.json [hunk 2]
```diff
@@ -10,6 +10,20 @@
...
+    "flag H1",
+    "flag G1",
+    "flag F1",
+    "clear E1",
+    "clear D1",
+    "clear F7",
+    "clear G7",
+    "clear E8",
+    "flag H7",
+    "clear H8"
```
Comment (EldadHellerman) [inline commands.json:27]: maybe have square bracket right after string?

**Review by EldadHellerman — COMMENTED** [review]
great overall

**Pull request merged**